src/components/cookie.tsx

```diff
@@ -41,12 +40,13 @@ interface Movie {
 
 */
 
-function Cookie(connect = false, url? : string,  methode? :"GET" | "POST" | "PUT" | "DELETE", data? : DataUser | CreateData | Movie | DeleteMovie) {
-     const validUrl = url ?? ""; // verifie si l'url n'est pas null ou undifinied
-    console.log(validUrl);
+function Cookie(connect = false, reqUrl? : string,  methode? :"GET" | "POST" | "PUT" | "DELETE", data? : DataUser | CreateData | Movie | DeleteMovie) {
```

Comment:
Pour pas que tu sois embeté avec reqUrl optionnel tu peux écrire
```suggestion
function Cookie(connect = false, reqUrl: string,  methode? :"GET" | "POST" | "PUT" | "DELETE", data? : DataUser | CreateData | Movie | DeleteMovie) {
```

Du coup plus besoin de tester `validUrl` car `reqUrl` sera toujours défini